src/main/java/fi/dy/masa/malilib/util/InventoryUtils.java

```diff
@@ -472,4 +645,86 @@ public static Inventory getAsInventory(DefaultedList<ItemStack> items)
 
         return inv;
     }
+
```

Comment:
Moved here from Litematica, and expanded on it's features to be able to add Data Components.